docs/source/deploying.rst

```diff
@@ -0,0 +1,100 @@
+.. _deploying:
+
+Deployment
+==========
+
+Start Finch PyWPS service
+-------------------------
+
+After successful installation, you can start the service using the ``finch`` command-line:
+
+.. code-block:: console
+
+   $ finch --help  # show help
+   $ finch start  # start service with default configuration
+
+   OR
+
+   $ finch start --daemon  # start service as daemon
+   loading configuration
+   forked process id: 42
+
+The deployed WPS service is available by default on port 5000:
+
+http://localhost:5000/wps?service=WPS&version=1.0.0&request=GetCapabilities.
+
+.. note:: Remember the process ID (PID) so you can stop the service with ``kill PID``.
```

Comment:
I know you only copied this, but doesn't  `finch` poop a `pywps.pid` file so that we don't have to remember a pid ?

Reply:
It does! Maybe I should make reference to it instead. Noted.